src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
+</html>
```

Comment:
leave one empty line at the end of the file
https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/